realm/services/google_oauth/incoming_webhooks/oauth/source.js

```diff
@@ -15,17 +15,13 @@ exports = async function (payload, response) {
   const GOOGLE_TOKEN_ENDPOINT = "https://oauth2.googleapis.com/token";
 
   const SCOPES = [
-    "https://www.googleapis.com/auth/youtube",
+    //"https://www.googleapis.com/auth/youtube",
```

Comment:
Since these are commented out, can they be deleted?

Reply:
Maybe - I just wanted to check that all your scripts run successfully before removing the comments.